test-data/unit/check-except-star.test

```diff
@@ -0,0 +1,36 @@
+[case testExceptStarForbiddenBasic]
```

Comment:
I think these tests could fit into `check-python311.test`.

Consider cross checking these tests with CPython's test suite for break/continue/return in `except*` to make sure we're not missing any edge cases: https://github.com/python/cpython/blob/9d08423b6e0fa89ce9cfea08e580ed72e5db8c70/Lib/test/test_except_star.py#L51

Reply:
I cross-checked the CPython tests and the current tests cover most edge cases except a `finally:` block with `return` inside it.